### PR TITLE
Run4-hgx281X Update the test codes for HFNose

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Update the test codes for HFNose

#### PR validation:

Use test codes Geometry/ForwardCommonData/test

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@silviodonato @qliphy @cvuosalo @srimanob Can this be included in the pre6?

**Comment by srimanob** [comment]
Urgent

File: Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc [hunk 1]
```diff
@@ -26,7 +26,7 @@
 #include <unordered_set>
 #include <vector>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```
Comment (cvuosalo) [inline Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc:29]: Are the debug messages really needed in the master?

**Comment by cvuosalo** [comment]
+1

**Comment by cvuosalo** [comment]
@bsunanda Thanks for fixing this.

**Comment by cvuosalo** [comment]
@wpcarvalho @jan-kaspar FYI

File: Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc (hunk 1 above)
Comment (srimanob) [reply Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc:29]: @bsunanda 
Could you please confirm?

Comment (bsunanda) [reply Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc]: Sorry I am correcting for that

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by silviodonato** [comment]
merge
(it was already signed)

**Comment by cvuosalo** [comment]
+1

**Pull request merged**